contracts/interchain-token-service/tests/interchain_transfer.rs

```diff
@@ -63,3 +64,59 @@ fn interchain_transfer_send_fails_on_insufficient_balance() {
         &gas_token,
     );
 }
+
+#[test]
+fn interchain_transfer_fails_zero_amount() {
```

Comment:
we should review the missing code coverage separately to ensure everything is tested
https://axelarnetwork.atlassian.net/browse/AXE-7114